lib/umpire/web.rb

```diff
@@ -196,15 +203,25 @@ def create_aggregator(aggregation_method)
     end
 
     error do
-      Rollbar.error(sinatra_err.message, class: sinatra_err.class, at: :internal_error, request_id: request_id)
-      log(at: "internal_error", "class" => sinatra_err.class, message: sinatra_err.message)
+      log_exception(sinatra_err)
       status 500
       JSON.dump({"error" => "internal server error", "request_id" => request_id}) + "\n"
     end
 
+    def self.base_log_data
+      { scope: Thread.current[:scope] }
```

Comment:
you might want to `freeze` it so it doesn't accidentally get merged with something.